test/incompatible-shady.html

```diff
@@ -9,7 +9,9 @@
         window.ShadyCSS = {shimcssproperties: true};
       }
     </script>
-    <script src="../node_modules/wct-browser-legacy/browser.js"></script>
```

Comment:
did browser.js include a Symbol polyfill?

Reply:
browser.js didn't directly load a polyfill, but it did load several other scripts directly:
 - `stacky/browser.js`
 - `async/lib/async.js`
 - `lodash/index.js`
 - `mocha/mocha.js`
 - `chai/chai.js`
 - `@polymer/sinonjs/sinon.js`
 - `sinon-chai/lib/sinon-chai.js`
 - `accessibility-developer-tools/dist/js/axs_testing.js`
 - `@polymer/test-fixture/test-fixture.js`

I don't know if the situation described in https://github.com/Polymer/lit-html/pull/577#issuecomment-431556272 is ameliorated coincidentally by the wct-browser-legacy dependency set and loading behavior somehow causing babel-polyfill to load before webcomponentsjs.  Investigating whether this is the root cause and/or whether to fix in wct-mocha or wait for webcomponentsjs update to address.

Reply:
From my observations, when webcomponents polyfill is placed before browser.js, error happens. Otherwise, no errors. Have not found a root issue thogh.